api/libretime_api/storage/models/file.py

```diff
@@ -2,82 +2,122 @@
+    cue_in = models.DurationField(db_column="cuein", blank=True, null=True)
+    cue_out = models.DurationField(db_column="cueout", blank=True, null=True)
+    silan_check = models.BooleanField(blank=True, null=True)  # Unused, to remove ?
+
+    # https://github.com/libretime/libretime/issues/1366
+    track_type = models.CharField(max_length=16, blank=True, null=True)
+
+    artwork = models.CharField(max_length=512, blank=True, null=True)
+
+    # Audio
     bit_rate = models.IntegerField(blank=True, null=True)
     sample_rate = models.IntegerField(blank=True, null=True)
     format = models.CharField(max_length=128, blank=True, null=True)
+    channels = models.IntegerField(blank=True, null=True)
     length = models.DurationField(blank=True, null=True)
+
+    # Metadata
+    bpm = models.IntegerField(blank=True, null=True)
+    replay_gain = models.DecimalField(
+        max_digits=8,
+        decimal_places=2,
+        blank=True,
+        null=True,
+    )
+
+    name = models.CharField(max_length=255)  # Unused ?
+    description = models.CharField(
+        max_length=512,
+        blank=True,
+        null=True,
+    )  # Replace comment ?
+
+    artist_name = models.CharField(max_length=512, blank=True, null=True)
     album_title = models.CharField(max_length=512, blank=True, null=True)
+    track_title = models.CharField(max_length=512, blank=True, null=True)
     genre = models.CharField(max_length=64, blank=True, null=True)
-    comments = models.TextField(blank=True, null=True)
     year = models.CharField(max_length=16, blank=True, null=True)
     track_number = models.IntegerField(blank=True, null=True)
-    channels = models.IntegerField(blank=True, null=True)
-    url = models.CharField(max_length=1024, blank=True, null=True)
-    bpm = models.IntegerField(blank=True, null=True)
-    rating = models.CharField(max_length=8, blank=True, null=True)
-    encoded_by = models.CharField(max_length=255, blank=True, null=True)
     disc_number = models.CharField(max_length=8, blank=True, null=True)
-    mood = models.CharField(max_length=64, blank=True, null=True)
-    label = models.CharField(max_length=512, blank=True, null=True)
+    # https://github.com/libretime/libretime/issues/526
+    comment = models.TextField(db_column="comments", blank=True, null=True)
     composer = models.CharField(max_length=512, blank=True, null=True)
+    conductor = models.CharField(max_length=512, blank=True, null=True)
+    copyright = models.CharField(max_length=512, blank=True, null=True)
+    language = models.CharField(max_length=512, blank=True, null=True)
     encoder = models.CharField(max_length=64, blank=True, null=True)
-    checksum = models.CharField(max_length=256, blank=True, null=True)
-    lyrics = models.TextField(blank=True, null=True)
+    encoded_by = models.CharField(max_length=255, blank=True, null=True)
+    isrc_number = models.CharField(max_length=512, blank=True, null=True)
```

Comment:
Many fields could be renamed to something simpler, when obvious, here for example `isrc_number` => `irsc`